doc/getting_started/getting_started.rst

```diff
@@ -46,6 +62,29 @@ by Zephyr:
 Most of the examples in the Zephyr documentation use ``ninja`` as a build tool,
 but you should be able to use any generator on any of the examples listed.
 
+.. _getting_started_local_cmake:
+
+Development Environment Configuration File
+******************************************
+
+The development environment configuration file :file:`local.cmake`
+should be created to manage user-specific configurations such as
+install paths.
+
+It is a CMake build script that is executed early in the configuration
```

Comment:
Change to:

    A CMake build script is used early in the configuration process
    to replace all uses of environment variables, except for ``ZEPHYR_BASE``.
    The following sections define where this :file:`local.cmake` file lives
    and what to put in it.

    The :file:`local.cmake` file location depends on the platform: